ecommerce/backend/getflix/settings.py

```diff
@@ -26,9 +26,14 @@
     'HOST': os.environ.get('DB_HOST'),
     'USER': os.environ.get('DB_USER'),
     'PASSWORD': os.environ.get('DB_PASSWORD'),
+    'EMAIL_BACKEND': os.environ.get('EMAIL_BACKEND'),
+    'MAILER_EMAIL_BACKEND': os.environ.get('MAILER_EMAIL_BACKEND'),
+    'EMAIL_HOST': os.environ.get('EMAIL_HOST'),
+    'EMAIL_HOST_PASSWORD': os.environ.get('EMAIL_HOST_PASSWORD'),
+    'EMAIL_HOST_USER': os.environ.get('EMAIL_HOST_USER'),
+    'DEFAULT_FROM_EMAIL': os.environ.get('DEFAULT_FROM_EMAIL')
```

Comment:
For those, I think we need to configure our backend deploy script and add encrypted version of email password etc. to repo.

Reply:
Merging to master after those variables are configured would be better for deployment purposes